### DISCUSSION TITLE
open url in iframe

### DISCUSSION THREAD

**Discussion by overstartup** [OrchardCMS/OrchardCore#13857] [Q&A]
I need to open external url in my project by iframe. but i get this error:

`Refused to frame 'https://www.w3schools.com/' because an ancestor violates the following Content Security Policy directive: "frame-ancestors 'self' https://mycourses.w3schools.com".`

how can i solve this issue?

**Comment by hishamco** [comment]
Did you enable Security module?

**Reply by overstartup** [reply]
yes, i did, but i don't know how to config it.

**Comment by ns8482e** [comment]
You can't ! If owning website server (here www.w3schools.com ) has explicitly denied do that by having ` frame-ancestors 'self'`  
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors

**Reply by overstartup** [reply]
it should be none?
 frame-ancestors 'none'
